### ISSUE TITLE
[Suggestion] Show all combinations next to keypad on larger screens

### ISSUE THREAD

**Issue by sol3uk** [SavageCore/pd3-vault-cracker#11]
Just a suggestion for an ease of use improvement. I can have a crack at it if you like but I'm not 100% on my svelte.
When I use the app I have to keep the "all combinations" modal open on top to see them all, I'm not always able to tab out and click the next button. And If I'm using my second screen or tablet, the screen has so much free space, so I would suggest the option to display the contents of the modal on the right or left of the keypad like so:
![image](https://github.com/SavageCore/pd3-vault-cracker/assets/26093813/bc7eae71-bb4a-448b-bcb9-844039eba949)
Maybe offer the user a "View" option that can be toggled? And I'd probably keep both of the elements centred to keep it looking nice.

Just a suggestion so feel free to close it off, just thought it'd be a nice little addition. 
Good job on the app so far, was really surprised when I found someone had already made one!

**Comment by SavageCore** [comment]
Good idea! I think a customisable view will be best defaulting to what we have now. Feel free to take a stab at it, doubt I'll get to it until tomorrow night at the earliest.

**Comment by sol3uk** [comment]
I'm gonna be away for a while so won't be able to have a crack right now. Will try when I get back if you haven't already beat me to it!
